security/tpm/src/tpm.ts

```diff
@@ -303,7 +310,9 @@ export class TpmSecurityClient  {
   private _getPropsAndHashAlg( callback: (algorithm: TPM_ALG_ID, properties: tss.TPML_TAGGED_TPM_PROPERTY) => void): void {
     const idKeyHashAlg: TPM_ALG_ID = (<tss.TPMS_SCHEME_HMAC>(<tss.TPMS_KEYEDHASH_PARMS>this._idKeyPub.parameters).scheme).hashAlg;
 
-    this._tpm.GetCapability(tss.TPM_CAP.TPM_PROPERTIES, TPM_PT.INPUT_BUFFER, 1, (caps: tss.GetCapabilityResponse) => {
+    this._tpm.GetCapability(tss.TPM_CAP.TPM_PROPERTIES, TPM_PT.INPUT_BUFFER, 1, (err: tss.TpmError, caps: tss.GetCapabilityResponse) => {
```

Comment:
>this._tpm.GetCapability [](start = 4, length = 23)

Should this _privates signature be changed to propagate through the error from the GetCapability?